nservicebus/transports/guidance.md

```diff
@@ -0,0 +1,85 @@
+
+The best option is to store the queues in the same catalog as the business data. Schemas can be used to make maintenance easier and allow for finer-grained security. 
+
+#### Security
+
+For best results, both in terms of security and maintainability, each endpoint should run in context of a different account. Each such account should own three schemas on the database level
+ * the data schema (e.g. `sales`) for which it has exclusive write access. Other accounts might optionally have read access to this schema.
+ * the incoming queue schema (e.g. `sales-incoming`) for which it has exclusive `DELETE` access. Other accounts should have `INSERT` permissions to this schema
+ * the private queue schema (e.g. `sales-nsb`) for which it has exclusive access. No other accounts should have any permissions to this schema.
+
+This approach ensures that endpoints are allowed to send messages to each other and (optionally) read each other's data. They are not allowed to receive other endpoint's messages nor modify their data directly.
+
+### Medium to large
+
+For larger systems it usually makes more sense to have separate databases for each service or bounded context. It is also not unusual to see NoSQL databases in at least some of the services. The SQL databases might be hosted in a single instance of SQL Server or in separate instances depending on the IT policy.
+
+Such systems consist of tens to hundreds of endpoints. Some of the endpoints might be scaled out for higher throughput. 
+
+The best option is to have a dedicated catalog for the queues. This approach allows to use different scaling up strategies for the queue catalog. It also allows to use a dedicated back up policy for the queues (because data in queues has much more ephemeral). Unfortunately it leaves a problem of how to ensure that data modifications and queue operations are atomic and isolated from each other. 
+
+One option is to use Microsoft Distributed Transaction Coordinator and distributed transactions (NServiceBus with SQL Server transport is DTC-enabled by default). In this mode the message receive transaction and all data modifications that result from processing the message are part of one distributed transaction that spans two SQL catalogs.
+
+Another options is to use Outbox, which is a unique feature of NServiceBus, that provides exactly-once processing semantics over an at-least-once infrastructure. In this mode each individual database has to include the outbox table. Outgoing messages are stored in that table in the same transaction as the business data is modified and, only after that transaction commits successfully, they are dispatched to the destination. 
+
+#### Scaling up
```

Comment:
This is a placeholder. I think we should evolve the guidance in this direction (to include it) but I think we can release it initially without it.